docs/source/examples/checks/performance/simple_model_comparison.ipynb

```diff
@@ -5,118 +5,84 @@
    "id": "840ea662",
```

Comment:
<p>"If the user model achieves less or similar performance to the simple model, than there is probably an issue with the model (for example, the model may not be treating class imbalance properly). " -&gt;</p><p>"If the user model achieves less or a similar score to the simple model, this is an indicator for a possible problem with the model (e.g. it wasn't trained properly)"</p><p> </p><p>"There are 3 types of simple models exists in the check. " -&gt; grammar inconsistency</p><p>"Contstant" -&gt; "Constant"</p><p> </p><p>"&nbsp;In regression is mean value, in classification the most common value." -&gt; grammar (e.g. add "is equal to the...")</p><p>Random - select by random according to which distribution?</p><p>"Runs" -&gt; trains a simple decision tree model...?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepchecks/deepchecks/pull/613/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>